qcore/utils.py

```diff
@@ -200,7 +209,28 @@ def load_py_cfg(f_path):
     :return: dict of parameters
     """
     with open(f_path) as f:
-        module = imp.load_module('params', f, f_path, ('.py', 'r', imp.PY_SOURCE))
+        module = imp.load_module("params", f, f_path, (".py", "r", imp.PY_SOURCE))
         cfg_dict = module.__dict__
 
     return cfg_dict
+
+
+def compare_versions(version1, version2):
+    parts1 = version1.split(".")
```

Comment:
Comment to state what each return code means.

Actually, an enum would be pretty cool.